templates/default/070-components/UI-framework/MainControls/_ui-component_mode_info.scss

```diff
@@ -1,61 +1,101 @@
 $il-mode-info-border: 3px solid $il-mode-info-color;
-$il-mode-info-height: 30px;
+$il-mode-info-height: 35px;
 $il-mode-info-text-color: white;
 $il-mode-info-zindex: 1010;
-$il-mode-info-shadow: 0px 2px 2px 0px rgba(128, 128, 128, 1);
-
-.il-mode-info {
-  border-top: $il-mode-info-border;
-  border-left: $il-mode-info-border;
-  border-right: $il-mode-info-border;
-  text-align: center;
+$il-mode-info-shadow: 20px 10px 30px rgba(0, 0, 0, .15);
 
-  z-index: $il-mode-info-zindex;
+.c-mode-info {
+  display: flex;
   position: absolute;
-  height: 100%;
+  z-index: $il-mode-info-zindex;
   width: 100%;
-  pointer-events: none;
-}
+  align-items: start;
+  justify-content: center;
+  text-align: center;
 
-.il-mode-info + div {
-  border: $il-mode-info-border;
-  border-top: 0 none;
-  
-  @media only screen and (max-width: $il-grid-float-breakpoint-max) {
-    padding-top: $il-mode-info-height;
+  &__pageframe {
+    border: $il-mode-info-border;
+    height: 100%;
+    position: fixed;
+    width: 100%;
+    pointer-events: none;
+    z-index: $il-mode-info-zindex;
   }
-}
 
+  &__content {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $il-mode-info-color;
+    border-bottom-left-radius: $il-border-radius-secondary-large;
+    border-bottom-right-radius: $il-border-radius-secondary-large;
+    -webkit-box-shadow: $il-mode-info-shadow;
+    -moz-box-shadow: $il-mode-info-shadow;
+    box-shadow: $il-mode-info-shadow;
+    padding: $il-padding-xs-vertical $il-padding-xxxlarge-horizontal;
+    gap: $il-padding-xxxlarge-horizontal;
+    min-height: $il-mode-info-height;
+    min-width: 20%;
+    max-width: 50%;
 
-.il-mode-info span.il-mode-info-content {
-  margin: auto;
-  background-color: $il-mode-info-color;
-  padding: $il-padding-base-vertical $il-padding-base-horizontal $il-padding-base-vertical $il-padding-base-horizontal;
-  -webkit-box-shadow: $il-mode-info-shadow;
-  -moz-box-shadow: $il-mode-info-shadow;
-  box-shadow: $il-mode-info-shadow;
+    @media only screen and (max-width: $il-grid-float-breakpoint-max) {
+      border-radius: 0;
+      width: 100%;
+      min-height: $il-standard-page-small-header-height;
+      max-width: 100%;
+    }
+  }
 
-   pointer-events: all;
-}
+  &__label,
+  &__close {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    pointer-events: all;
+    position: relative;
+  }
 
-.il-mode-info span {
-  color: $il-mode-info-text-color;
-  font-size: $il-font-size-large;
-}
+  &__label {
+    @include il-multi-line-cap-mixin(1);
+    flex-grow: 1;
+  }
 
-@media only screen and (max-width: $il-grid-float-breakpoint-max) {
-  .il-mode-info {
-	margin-bottom: 0;
+  &__close {
+
+  }
+
+  &__mobile-padding {
+    @media only screen and (max-width: $il-grid-float-breakpoint-max) {
+        height: $il-standard-page-small-header-height;
+    }
   }
 
-  span.il-mode-info-content {
-	display: block;
-	width: 100%;
+  * {
```

Comment:
Is it really necessary to be so general?

Reply:
its just a bit shorter